app/types/module.py

```diff
@@ -22,6 +23,7 @@ def __init__(
         ]
         | None = None,
         registred_topics: list[Topic] | None = None,
+        permissions: type[ModulePermissions] | None = None,
     ):
         """
         Initialize a new Module object.
```

Comment:
```suggestion
        Initialize a new Module object.
        
```
fixes the docstring (which should be sompleted with the new permissions param)